apps/server/src/mikro-orm.config.ts

```diff
@@ -0,0 +1,36 @@
+import type { MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs/typings';
+import { ALL_ENTITIES } from '@shared/domain/entity';
+import { FileEntity } from '@modules/files/entity';
+import { FileRecord } from '@modules/files-storage/entity';
+import { Dictionary, IPrimaryKey } from '@mikro-orm/core';
+import { NotFoundException } from '@nestjs/common';
+import { DB_PASSWORD, DB_URL, DB_USERNAME } from './config';
+
+export const mikroOrmConfig: MikroOrmModuleSyncOptions = {
+	// TODO repeats server module definitions
+	type: 'mongo',
+	clientUrl: DB_URL,
+	password: DB_PASSWORD,
+	user: DB_USERNAME,
+	entities: [...ALL_ENTITIES, FileEntity, FileRecord],
+	allowGlobalContext: true,
+	findOneOrFailHandler: (entityName: string, where: Dictionary | IPrimaryKey) =>
+		/* istanbul ignore next */
+		new NotFoundException(`The requested ${entityName}: ${JSON.stringify(where)} has not been found.`),
+	migrations: {
+		tableName: 'migrations', // name of database table with log of executed transactions
+		path: './dist/apps/server/src/migrations/mikro-orm', // path to the folder with migrations
+		pathTs: './apps/server/src/migrations/mikro-orm', // path to the folder with TS migrations (if used, we should put path to compiled files in `path`)
+		glob: '!(*.d).{js,ts}', // how to match migration files (all .js and .ts files, but not .d.ts)
+		transactional: false, // wrap each migration in a transaction
```

Comment:
transations need MongoBD replica - local / dev cluster does not work
perhaps TODO for staging / prod